### PR TITLE
Nav style

### PR DESCRIPTION
**Pull request by nogainbar**
Changed colour of nav and added Healthlocker logo

### CODE REVIEW

File: web/static/css/app.css [hunk 1]
```diff
@@ -30,3 +39,50 @@
...
+
+.hl-pink {
+  color: #EC4067;
+}
+
+.hl-bg-pink {
+  background-color: #EC4067;
+}
+
+.hl-darkBlue {
```
Comment (tonydaly) [inline web/static/css/app.css]: Could we stick with kebab-case rather than camelCase.

File: web/static/css/app.css [hunk 2]
```diff
@@ -30,3 +39,50 @@
...
+  color: #242F4F;
+}
+
+.hl-bg-darBlue {
```
Comment (tonydaly) [inline web/static/css/app.css]: typo?

File: web/templates/layout/app.html.eex [hunk 3]
```diff
@@ -16,6 +16,14 @@
   </head>
 
   <body class="w-100 sans-serif bg-white">
+
+    <header class="hl-bg-aqua w-100 ph3 pv3 pv4-ns ph4-m ph5-l tc">
+      <nav class="f4 fw3 tracked">
+        <i id="open-nav" class="fa fa-bars dim white dib fl" aria-hidden="true"></i>
+        <a href="/"><%= img_tag("/images/logo.svg", class: "w-70") %></a>
```
Comment (tonydaly) [inline web/templates/layout/app.html.eex]: I've got a horrible feeling we might need to support IE8, if so we'll need an SVG fallback.

File: web/static/css/app.css [hunk 4]
```diff
@@ -30,3 +39,50 @@
 .bg-fly-high {
   background-image: linear-gradient(to top, #48c6ef 0%, #6f86d6 100%);
 }
+
+
+/*color pallette*/
```
Comment (tonydaly) [inline web/static/css/app.css]: "color palette"

File: web/templates/layout/app.html.eex (hunk 3 above)
Comment (nogainbar) [inline web/templates/layout/app.html.eex]: Cool, I didnt do a fallback, I just changed the image to png instead. 

Are we doing a check for whether its IE8 or not? I can apply the trick of:
`<svg width="96" height="96">
  <image xlink:href="image.svg" src="image.png" width="96" height="96" />
</svg>`

but not sure how to do it in Phoenix...

**Comment by nogainbar** [comment]
Applied all changes. :)

File: web/static/css/app.css (hunk 1 above)
Comment (nogainbar) [inline web/static/css/app.css]: sure! that slippery cheeky camelCase.. ;)

**Comment by katbow** [comment]
@nogainbar please fix conflicts so this can be merged in ⭐️

**Comment by nelsonic** [comment]
@tonydaly please confirm the changes made by @nogainbar are OK. and merge the PR. thanks! ❤️

File: web/templates/login/index.html.eex [hunk 5]
```diff
@@ -1,25 +1,34 @@
...
-  <h3 class="tc">Log in</h3>
+  <h3 class="hl-darkBlue tc">Log in</h3>
   <section class="mw7 center">
-    <%= form_for @conn, login_path(@conn, :create), [as: :login, class: "mw6 bg-white br2 br--bottom shadow-1--bottom pa3 mv3 ba b--black-10 center"], fn f -> %>
+    <div class="bb b--black-20 pb4 mt4">
+      <span class="fl mt0">Not an existing user?</span>
+      <a class="link dim fr mt0 db" href="/users/new">Sign up</a>
+    </div>
+
+    <!-- form -->
```
Comment (tonydaly) [inline web/templates/login/index.html.eex]: Hmm I'm not a fan of these types of comments, but fine if the team thinks they're useful.

File: web/templates/login/index.html.eex [hunk 6]
```diff
@@ -1,25 +1,34 @@
...
+        <a href="#0" class="f5 link dim db">Forgotten your password?</a>
       </div>
 
-      <div class="lh-copy mt3">
-        <a href="#0" class="f6 link dim black-60 db">Sign up</a>
-        <a href="#0" class="f6 link dim black-60 db">Forgot your password?</a>
+      <div>
+        <%= submit "Log in", class: "mt4 f5 link dim br-pill ph3 pv2 dib black-60 hl-bg-grey pointer fr w-40" %>
       </div>
+
```
Comment (tonydaly) [inline web/templates/login/index.html.eex]: Lots of extra space here, any particular reason?

File: .gitignore [hunk 7]
```diff
@@ -3,6 +3,7 @@
 /db
 /deps
 /*.ez
+.DS_Store
```
Comment (tonydaly) [inline .gitignore]: This is ok, but you should also add it to your global `.gitignore` as not all projects will have this.

File: web/templates/login/index.html.eex (hunk 5 above)
Comment (nogainbar) [inline web/templates/login/index.html.eex]: Took it out (im not sure I put it in..?) :-o

File: web/templates/login/index.html.eex (hunk 6 above)
Comment (nogainbar) [inline web/templates/login/index.html.eex]: took all extra space out :)

File: .gitignore (hunk 7 above)
Comment (nogainbar) [inline .gitignore]: So weird!! ive put it in the gitignore!! not sure what going on...

**Pull request merged**